src/game-support/mb-wfas.md

```diff
@@ -1,7 +1,7 @@
 # Mount & Blade: With Fire & Sword
 <!-- script:Aliases [] -->
```

Comment:
Can you add an alias here too?
- `Mount and Blade: With Fire and Sword`
- `Mount & Blade With Fire & Sword`
- `Mount and Blade With Fire and Sword`